From e98b9228b5ecb6116aabc7e2c4f347567c7f1eee Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Tue, 12 Dec 2023 13:16:04 +0100
Subject: [PATCH] arm64: defconfig: add CONFIG_IGC in
 fragment-02-defconfig-addons

Enable IGC Ethernet to use PTM support.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: Iaa9f29e10e30c131a9e0a2fdca4c125405533532
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/361775
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index ec06caa400b7..7de0e095681f 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -27,6 +27,7 @@ CONFIG_HDMI=y
 CONFIG_I2C_RPMSG=m
 CONFIG_I2C_STM32F7=y
 CONFIG_I3C=y
+CONFIG_IGC=m
 CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
 CONFIG_IIO_STM32_TIMER_TRIGGER=m
 CONFIG_JFFS2_FS=y
-- 
2.39.5

